dev_guide/builds.adoc

```diff
@@ -110,10 +110,14 @@ code. If your application exists inside a sub-directory, you can override the
 default location (the root folder) using this field.
 ====
 
-[[using-the-sti-environment-file]]
-== STI Environment File
+[[configuring-the-sti-environment]]
+== STI Environment
 
-link:../image_writers_guide/sti.html[STI] enables you to set environment values
+There are two ways to make environment variables available to the link:../image_writers_guide/sti.html[STI] 
```

Comment:
This is a broken link (old pre-restructure path). Can you update during rebase? The current target is `../architecture/core_objects/builds.html#sti-build`, if you want to use that. Or if you still want that original topic in particular, `../creating_images/sti.html`.